app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -57,53 +51,91 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
-     --network=<div contenteditable="true">{NETWORK_NAME}</div> \
+    ```sh
+    docker run -d --name kong-database \
+     --network=kong-net \
      -p 5432:5432 \
-     -e "POSTGRES_USER=<div contenteditable="true">{DATABASE_USER}</div>" \
-     -e "POSTGRES_DB=<div contenteditable="true">{DATABASE_NAME}</div>" \
-     -e "POSTGRES_PASSWORD=<div contenteditable="true">{DATABASE_PASSWORD}</div>" \
+     -e "POSTGRES_USER=kong" \
+     -e "POSTGRES_DB=kong" \
+     -e "POSTGRES_PASSWORD=kongpass" \
      postgres:9.6
-   </code></pre>
+    ```
+
+    * `POSTGRES_USER` and `POSTGRES_DB`: Set these values to `kong`. This is
+    the default value that Kong Gateway expects.
+    * `POSTGRES_PASSWORD`: Set the database password to any string.
+
+    In this example, the Postgres container named `kong-database` can
+    communicate with any containers on the `kong-net` network.
 
 1. Prepare the Kong database:
+
 {% capture migrations %}
-<pre><code>docker run --rm --network=<div contenteditable="true">{NETWORK_NAME}</div> \
+{% navtabs codeblock %}
+{% navtab Kong Gateway %}
+```sh
+docker run --rm --network=kong-net \
  -e "KONG_DATABASE=postgres" \
- -e "KONG_PG_HOST=<div contenteditable="true">{PG_CONTAINER_NAME}</div>" \
- -e "KONG_PG_PASSWORD=<div contenteditable="true">{CONTAINER_PASSWORD}</div>" \
- -e "KONG_PASSWORD=<div contenteditable="true">{PASSWORD}</div>" \
- <div contenteditable="true">{TAG_NAME}</div> <div contenteditable="true">{DATABASE_NAME}</div> migrations bootstrap</code></pre>
+ -e "KONG_PG_HOST=kong-database" \
+ -e "KONG_PG_PASSWORD=kongpass" \
+ -e "KONG_PASSWORD=test" \
+kong/kong-gateway kong migrations bootstrap
+```
+{% endnavtab %}
+{% navtab Kong Gateway (OSS) %}
+```sh
+docker run --rm --network=kong-net \
+ -e "KONG_DATABASE=postgres" \
+ -e "KONG_PG_HOST=kong-database" \
+ -e "KONG_PG_PASSWORD=kongpass" \
+kong kong migrations bootstrap
+```
+{% endnavtab %}
+{% endnavtabs %}
 {% endcapture %}
 {{ migrations | indent | replace: " </code>", "</code>" }}
 
+    Where:
+    * [`KONG_DATABASE`](/gateway/{{page.kong_version}}/reference/configuration/#database)
+     is the database type, or `off` if setting up a data plane for [hybrid mode](/gateway/{{page.kong_version}}/plan-and-deploy/hybrid-mode/hybrid-mode-setup).
+    * [`KONG_PG_HOST`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings)
+    is the name of the Postgres Docker container that is communicating over the
+    `kong-net` network, from the previous step.
+    * [`KONG_PG_PASSWORD`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings)
+    The password that you set when bringing up the Postgres container in the
+    previous step.
+    * `KONG_PASSWORD` (Enterprise only): The default password for the admin
+    super user for Kong Gateway.
+    * `kong/kong-gateway kong migrations bootstrap` or `kong kong migrations bootstrap`:
```

Comment:
We can remove the image name here as they're not expected to copy/paste from the explanation. Showing both may lead to questions about when to use `kong` or `kong/kong-gateway`

Reply:
My goal here was to clear up the confusion around `kong kong` - the question about what this means and why it sometimes works and sometimes doesn't has floated around the internet for a long time. So the explanation is useful, but I can leave a placeholder for the image name, eg `<image-name> kong migrations bootstrap`